File: Elixir/0001.ex
defmodule MultiplesSum do
  def sum_of_multiples(limit) do
    Enum.reduce(0..(limit-1), 0, fn i, acc ->
      if rem(i, 3) == 0 or rem(i, 5) == 0 do
        acc + i
      else
        acc
      end
    end)
  end
end

# Usage
IO.puts("The sum of all multiples of 3 or 5 below 1000 is: #{MultiplesSum.sum_of_multiples(1000)}")

File: Erlang/0001.erl
-module(sum_multiples).
-export([main/1]).

% Entry point for the escript
main(Args) ->
    Limit = 1000,
    Sum = sum_of_multiples(Limit),
    io:format("The sum of all multiples of 3 or 5 below ~p is: ~p~n", [Limit, Sum]).

% Function to calculate the sum of multiples of 3 or 5
sum_of_multiples(Limit) when Limit > 0 ->
    Sum = lists:sum(
        [X || X <- lists:seq(1, Limit - 1), (X rem 3 =:= 0 orelse X rem 5 =:= 0)]
    ),
    Sum.